cves/kernel/CVE-2017-16911.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  The Virtual Host Controller Interface driver in the Linux Kernel, which can 
+  allow users to use a USB device remotely via IP, allows attackers to leak 
+  kernel memory addresses within a status file or via debug output. The driver 
+  was implemented this way as a convenient way of obtaining the IP address from 
+  a memory address.
```

Comment:
I really like how you include the note about why the driver was initially implemented this way. It gives better context to the reader that the system was designed this way on purpose, but it just has unintended consequences.

Reply:
I agree this description makes the vulnerability very easy to understand (especially after learning xml dtd) and i agree it really does give better context and reinforces the idea that scope creep/adding features can infact lead to security issues

Reply:
Thank you both